PKGBUILD

```diff
@@ -77,19 +77,34 @@ prepare() {
   export TF_IGNORE_MAX_BAZEL_VERSION=1
   export TF_MKL_ROOT=/opt/intel/mkl
   export NCCL_INSTALL_PATH=/usr
-  export GCC_HOST_COMPILER_PATH=/usr/bin/gcc-9
-  export HOST_C_COMPILER=/usr/bin/gcc-9
-  export HOST_CXX_COMPILER=/usr/bin/g++-9
+  # export GCC_HOST_COMPILER_PATH=/usr/bin/gcc-9
+  export GCC_HOST_COMPILER_PATH=/opt/rocm/hip/bin/hipcc
+  # export HOST_C_COMPILER=/usr/bin/gcc-9
+  export HOST_C_COMPILER=/opt/rocm/hip/bin/hipcc
+  # export HOST_CXX_COMPILER=/usr/bin/g++-9
+  export HOST_CXX_COMPILER=/opt/rocm/hip/bin/hipcc
   export TF_CUDA_CLANG=0  # Clang currently disabled because it's not compatible at the moment.
   export CLANG_CUDA_COMPILER_PATH=/usr/bin/clang
   export TF_CUDA_PATHS=/opt/cuda,/usr/lib,/usr
-  export TF_CUDA_VERSION=$(/opt/cuda/bin/nvcc --version | sed -n 's/^.*release \(.*\),.*/\1/p')
-  export TF_CUDNN_VERSION=$(sed -n 's/^#define CUDNN_MAJOR\s*\(.*\).*/\1/p' /usr/include/cudnn.h)
+  export TF_CUDA_VERSION=$([ -e /opt/cuda ] && /opt/cuda/bin/nvcc --version | sed -n 's/^.*release \(.*\),.*/\1/p' || echo "")
+  export TF_CUDNN_VERSION=$([ -e /usr/include/cudnn.h ] && sed -n 's/^#define CUDNN_MAJOR\s*\(.*\).*/\1/p' /usr/include/cudnn.h || echo "")
```

Comment:
Why is this changed? Why do we even care about it?

Reply:
It's just to avoid an error, but I guess it is not crutial

Reply:
If it's to avoid an error that seems crucial to me, Does the build still go on? If so then lets leave it as is.

Reply:
Just to clarify, the error is that it tries to read cuda attributes and fails. The build carries on though.

Reply:
I see kk, then it makes sense to keep this change. Can you add a comment about these two lines with a short description however?

Reply:
Sure